ansible/roles/couchdb/tasks/clean.yml

```diff
@@ -4,6 +4,6 @@
 - name: remove CouchDB
   docker_container:
     name: couchdb
-    image: klaemo/couchdb
+    image: apache/couchdb
```

Comment:
This is a clean task to remove the container, I think the image name is ignored, we should try to remove this line

Reply:
Carlos,
Thanks for the review, I confirmed that "state: absent" doesn't do 'docker rmi', but 'docker rm'. So we should remove it.
--
Naohiro